README.md

```diff
@@ -1,66 +1,64 @@
-<!-- ABOUT THE PROJECT -->
 ## About The Project
 
+OpenShift LightSpeed is an AI powered assistant that runs on OpenShift and provides answers to product questions using backend LLM services.
 
-
-
-### Built With
-
-This project is built using IBM Watson machine learning.
-
-
-
-<!-- GETTING STARTED -->
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-
 * Python 3.11 or newer
 * Git and pip installed
-* BAM account
-    - BAM_API_KEY
-    - BAM_URL
+* An LLM api key, currently BAM(IBM's research environment) and OpenAI are offered as backends.
 
 ### Installation
 
-_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+#### Get API keys
 
+##### BAM provided LLM
 1. Get a BAM API Key at [https://bam.res.ibm.com](https://bam.res.ibm.com)
    * Login with your IBM W3 Id credentials.
    * Copy the API Key from the Documentation section.
      ![BAM API Key](docs/bam_api_key.png)
-2. BAM URL: https://bam-api.res.ibm.com
-3. Clone the repo
+2. BAM API URL: https://bam-api.res.ibm.com
+
+##### OpenAI provided LLM
+Get an [OpenAI api key](https://platform.openai.com/api-keys)
+
+#### Install the code
+1. Clone the repo
    ```sh
    git clone <project git>
    cd lightspeed-service
    ```
-4. Install python packages
+2. Install python packages
    ```sh
-   pip install -r requirements.txt
+   make install-deps
    ```
-5. OLS configuration is in YAML format. It is loaded from a file referred to by the OLS_CONFIG_FILE environment variable and defaults to olsconfig.yaml in the current directory. You can find
-a sample configuration in the olsconfig.yaml file in the top directory of this repository.
-API credentials are in turn loaded from their respective files, referred to by the credentials_path attributes in the configuration YAML. If these paths are relative, they are relative to the current working directory. Place your BAM API Key into the bam_api_key.txt file.
-6. Configure vector index store.
-   * Download local.zip from [releases](https://github.com/ilan-pinto/lightspeed-rag-documents/releases)
-   * Create vector index directory
-      ```sh
-       mkdir -p vector-db/ocp-product-docs
-      ```
-   * Unzip local.zip in vector-db/ocp-product-docs directory
-     ```sh
-     unzip -j <path-to-downloaded-file>/local.zip -d vector-db/ocp-product-docs
-     ```
-<!-- USAGE EXAMPLES -->
+
+#### Configure OLS
+
+OLS configuration is in YAML format. It is loaded from a file referred to by the `OLS_CONFIG_FILE` environment variable and defaults to `olsconfig.yaml` in the current directory. 
+You can find a sample configuration in the `olsconfig.yaml`` file in the top directory of this repository.  
+
+API credentials are in turn loaded from files specified in the config yaml by the `credentials_path` attributes in the configuration YAML. If these paths are relative, 
```

Comment:
we say 'the config yaml' twice in this sentence.

Reply:
fixed, thanks